src/main/java/org/gridsuite/directory/server/DirectoryController.java

```diff
@@ -53,6 +53,17 @@ public ResponseEntity<ElementAttributes> createElement(@PathVariable("directoryU
         return ResponseEntity.ok().contentType(MediaType.APPLICATION_JSON).body(service.createElement(elementAttributes, directoryUuid, userId));
     }
 
+    @PostMapping(value = "/directories/paths/elements", consumes = MediaType.APPLICATION_JSON_VALUE)
```

Comment:
I don't know. It's hard to come up with a coherent mapping for a call like that. I don't like /elements because it would clash too easily with any new endpoint we try to add.